test/jdk/java/lang/Float16/BasicFloat16ArithTests.java

```diff
@@ -375,11 +376,78 @@ private static void throwRE(String message) {
         throw new RuntimeException(message);
     }
 
+
+    private static void checkValueOfDouble() {
+        /*
+         * Check that double -> Float16 conversion rounds propertly
+         * around the widway point for each finite Float16 value.
+         */
+        for(int i = 0; i <= Short.MAX_VALUE; i++ ) {
+            // Start by just checking positive values...
```

Comment:
The test does, in fact, also check the negated value.
It doesn't check negative zero, though.

Maybe it's simpler to loop like so
```
        for(int i = Short.MIN_VALUE; i <= Short.MAX_VALUE; i++ ) {
```
and drop the logic for the negated value.

Reply:
Forget about this comment, except for the first line. The test is already complete